internal/rbd/rbd_util.go

```diff
@@ -192,28 +192,83 @@ type migrationVolID struct {
 	clusterID string
 }
 
-var supportedFeatures = map[string]imageFeature{
-	librbd.FeatureNameLayering: {
-		needRbdNbd: false,
-	},
-	librbd.FeatureNameExclusiveLock: {
-		needRbdNbd: false,
-	},
-	librbd.FeatureNameObjectMap: {
-		needRbdNbd: false,
-		dependsOn:  []string{librbd.FeatureNameExclusiveLock},
-	},
-	librbd.FeatureNameFastDiff: {
-		needRbdNbd: false,
-		dependsOn:  []string{librbd.FeatureNameObjectMap},
-	},
-	librbd.FeatureNameJournaling: {
-		needRbdNbd: true,
-		dependsOn:  []string{librbd.FeatureNameExclusiveLock},
-	},
-	librbd.FeatureNameDeepFlatten: {
-		needRbdNbd: false,
-	},
+var (
```

Comment:
@pkalever can we also document the justifying link as source code comment for below version mapping as referecnce ?

Reply:
There is nothing as such, I had to go to linux tree and find it myself :-(